### PR TITLE
Add Vincent to dev list

### LINKED ISSUE #190: added "Jhon Vincent Gupo" to the dev board

### PR DESCRIPTION
**Pull request by Ragnov**
closes #190

### CODE REVIEW

**Comment by Jansalvador1445** [comment]
Has a `Conflict`. please rebase.

**Comment by Ragnov** [comment]
rebase done, please check again. Thank you

File: .all-contributorsrc [hunk 1]
```diff
@@ -318,6 +318,21 @@
       "contributions": [
         "content"
       ]
+<<<<<<< HEAD
```
Comment (Jansalvador1445) [inline .all-contributorsrc:321]: This is a sign of conflict.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,34 @@
 # devs
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< .merge_file_a08748
+<<<<<<< .merge_file_a08308
```
Comment (Jansalvador1445) [inline README.md:7]: This is a sign of conflict.

**Comment by Jansalvador1445** [comment]
Not merging this one, maybe. Can you create another PR for this one?

Steps: 
1) git checkout master
2) git pull
3) git checkout -b "branch_name"
4) add codes
5) git add .
6) git commit -m "message"
7) git push

**Comment by Ragnov** [comment]
ok, i'll try what you suggested. Thank you

**Comment by Jansalvador1445** [comment]
No worries, In any case don't hesitate to ask for help saamin :D

**Pull request closed without merging**